native-windows-gui/src/layouts/mod.rs

```diff
@@ -7,3 +7,7 @@ pub use self::grid_layout::{GridLayout, GridLayoutInner, GridLayoutBuilder, Grid
 
 #[cfg(feature = "flexbox")]
 pub use self::flexbox_layout::{FlexboxLayout, FlexboxLayoutBuilder, FlexboxLayoutItem, FlexboxLayoutChildrenMut, FlexboxLayoutChildren};
+
+mod dyn_layout;
```

Comment:
Add `#[cfg(feature = "dynamic_layout")]` . You also need to add it to Cargo.toml under "features"